projects/epbs-implementation-in-prysm.md

```diff
@@ -0,0 +1,113 @@
+# EIP-7732: Enshrined Proposer-Builder Separation
+
+Implementation of a working Proof-of-Concept for [EIP-7732 or ePBS](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-7732.md).
+
+## Motivation
+
+Currently in the Ethereum, the Proposer-Builder Separation or PBS is handled by third-party software like mev-boost and an out-of-protocol relay network. The [Proposer-Builder Separation (PBS)](https://ethereum.org/en/roadmap/pbs) was introduced to segregate block proposing from block building. PBS democratizes access to MEV by enabling block proposers to sell their rights to construct a block, thereby creating a market for block builders. [mev-boost](https://github.com/flashbots/mev-boost), an out-of-protocol implementation of PBS built by [Flashbots](https://www.flashbots.net), has been dominantly adopted, accounting for approximately [90% of Ethereum blocks being produced](https://mevboost.pics).
```

Comment:
I would personally not write anything about MEV in a justification for ePBS, although people tend to disagree.

Reply:
I've added it just to provide a little bit of context and not as primary motivation